src/client/driver/driver.js

```diff
@@ -723,6 +771,10 @@ export default class Driver extends serviceUtils.EventEmitter {
                 this._handleGetWindows(msg, window);
             else if (msg.type === MESSAGE_TYPE.closeAllChildWindows)
                 this._handleCloseAllWindowsMessage(msg, window);
+            else if (msg.type === MESSAGE_TYPE.startToRestoreChildLink)
```

Comment:
Can we use `switch` instruction here?

Reply:
We can. Do you prefer `switch`? In case of switch we will need to add extra `return` statements.

Reply:
Yes, in this case the `switch` statement is more preferable.